packages/core/src/browser/style/tree.css

```diff
@@ -139,3 +139,11 @@
 .theia-tree-element-node {
     width: 100%
 }
+
+.theia-tree-folder-vertical-line .theia-TreeNodeContent {
+    border-left: 1px solid #f0f0f0;
```

Comment:
We should not use hardcoded color values, the following fails to display on light-themes:

<img width="1266" alt="Screen Shot 2020-01-17 at 4 39 18 PM" src="https://user-images.githubusercontent.com/40359487/72648090-1cb0d180-3948-11ea-90c0-f29b3f35ec2d.png">

Instead, we should use a builtin color which is updated per theme.
You can also take a look at #6475  for reference.

Reply:
You can take a look at the new coding guidelines regarding theming + colors: https://github.com/eclipse-theia/theia/wiki/Coding-Guidelines#theming

Reply:
I believe they use the following as a color:

```css
.monaco-tl-indent > .indent-guide {
  transition: border-color 0.1s linear;
}
```

https://github.com/microsoft/vscode/blob/0ae3b979d7847b1f8fdf57ba0aaf1d25484be827/src/vs/base/browser/ui/tree/media/tree.css#L25-L34

Reply:
I used `var(--theia-foreground)` for the color and it seems to work.